gnd/src/main/res/layout/map_move_feature_layout.xml

```diff
@@ -55,6 +55,7 @@
       app:layout_constraintBottom_toTopOf="@+id/confirm_button"
       app:layout_constraintEnd_toEndOf="parent"
       app:srcCompat="@drawable/ic_baseline_cancel_24"
+      android:onClick="@{() -> viewModel.cancelButtonClicked()}"
```

Comment:
Please rename these callbacks with `onCancelButtonClick()` and `onConfirmButtonClick()` or similar for consistency.